Parsers/ASimUserManagement/Parsers/ASimUserManagementMicrosoftSecurityEvent.yaml

```diff
@@ -0,0 +1,242 @@
+          | extend 
+              GroupIdType = iif(isnotempty(GroupId), "SID","")
+      ),(
+          SecurityEvent
+          | where not(disabled)
+          | where EventID in (4744, 4748, 4749, 4753, 4759, 4763)
+          | parse-kv EventData as 
+            (
+                TargetUserName:string,
+                TargetDomainName:string,
+                TargetSid:string,
+                SubjectUserSid:string,
+                AccountType:string,
+                SubjectLogonId:string,
+                SubjectDomainName:string,
+                SubjectUserName:string
+            ) 
+            with (regex=@'<Data Name="(\w+)">{?([^<]*?)}?</Data>')
+          | project-rename 
+              ActorOriginalUserType = AccountType,
+              ActorSessionId = SubjectLogonId,
+              ActorUserId = SubjectUserSid,
+              GroupDomain = TargetDomainName,
+              GroupId = TargetSid,
+              GroupName = TargetUserName
+          | extend GroupName = iff (GroupDomain == "", GroupName, strcat (GroupDomain, "\\" ,GroupName))
+          | parse-kv EventData as 
+              (
+                  MemberName:string,
+                  MemberSid:string
+              ) 
+              with (regex=@'<Data Name="(\w+)">{?([^<]*?)}?</Data>')
+          | project-rename 
+              TargetUserId = MemberSid,
+              TargetUsername = MemberName
+          | project TimeGenerated, Type, EventID, Computer, _ResourceId, _ItemId, GroupId, GroupName, ActorUserId, SubjectDomainName, SubjectUserName, ActorOriginalUserType, ActorSessionId, TargetUsername, TargetUserId
+          | extend 
+              GroupIdType = iif(isnotempty(GroupId), "SID","")
+      )
+  | lookup EventIDLookup on EventID
+  | invoke _ASIM_ResolveDvcFQDN ("Computer")
+  | lookup UserTypeLookup on ActorOriginalUserType
+  | project-rename 
+      DvcId = _ResourceId,
+      EventUid = _ItemId
+  | extend 
+      ActorDomain = SubjectDomainName,
+      ActorUserIdType = iif(isnotempty(ActorUserId), "SID",""),
+      ActorUsername = iff (SubjectDomainName == "", SubjectUserName, strcat (SubjectDomainName, '\\', SubjectUserName)),
+      Dvc = DvcHostname,
+      DvcIdType = iff (DvcId == "", "", "AzureResourceID"),
+      DvcOs = "Windows",
+      EventCount = int(1),
+      EventEndTime = TimeGenerated,
+      EventOriginalType = tostring(EventID),
+      EventProduct = 'Security Events',
+      EventResult = "Success",
+      EventSchema = "UserManagement",
+      EventSchemaVersion = "0.1.1",
+      EventSeverity = "Informational",
+      EventStartTime = TimeGenerated,
+      EventVendor = 'Microsoft',
+      Hostname = DvcHostname
+  | project-away Subject*
+  | extend
+      ActorUsernameType = _ASIM_GetUsernameType(ActorUsername),
+      GroupNameType = _ASIM_GetUsernameType(GroupName),
+      TargetUsernameType = _ASIM_GetUsernameType(TargetUsername),
```

Comment:
I can't make the difference between Regular and Admin based on the logs

Reply:
@KustoKing I think this one we can do since we are already setting ActorUserType. Can you have a look?
We have a helper function for this too: https://github.com/Azure/Azure-Sentinel/blob/master/ASIM/lib/functions/ASIM_GetUserType.yaml Can we use this instead of the table we are using?